### PR TITLE
feat(css/lints): Implement `selector-max-combinators` rule

### PR DESCRIPTION
**Pull request by g-plane**
**Description:**

Implemented [`selector-max-combinators`](https://stylelint.io/user-guide/rules/list/selector-max-combinators/) rule.

**BREAKING CHANGE:**

No.

### CODE REVIEW

File: crates/swc_css_lints/src/rules/mod.rs [hunk 1]
```diff
@@ -53,6 +55,9 @@ pub fn get_rules(LintParams { lint_config }: &LintParams) -> Vec<Box<dyn LintRul
         (&rules_config.color_no_invalid_hex).into(),
     ));
     rules.push(unit_no_unknown((&rules_config.unit_no_unknown).into()));
+    rules.push(selector_max_combinators(
```
Comment (kdy1) [inline crates/swc_css_lints/src/rules/mod.rs:58]: Oh wait, I think this will make linter always run regardless of config.
I missed it in previous review.

Comment (g-plane) [reply crates/swc_css_lints/src/rules/mod.rs:58]: You're right, but I think we should fix it in another PR?

Comment (kdy1) [reply crates/swc_css_lints/src/rules/mod.rs:58]: Yeah, you are right, thanks!

**Review by kdy1 — APPROVED** [review]
Thank you!

---

swc-bump:
 - swc_css_lints

**Pull request merged**